README.md

```diff
@@ -2,7 +2,12 @@
 
 [![Documentation Status](https://readthedocs.org/projects/microsoft-recommenders/badge/?version=latest)](https://microsoft-recommenders.readthedocs.io/en/latest/?badge=latest)
 
-## What's New (June 21, 2021)
+## What's New 
+### (July 22, 2021)
+
+We have changed the names of the folders which contain the source code, so that they are more informative. This implies that you will need to change any import statements that reference the recommenders package. Specifically, the folder `reco_utils` has been renamed to `recommenders` and its subfolders have been renamed according to [issue 1390](https://github.com/microsoft/recommenders/issues/1390).  
+
+### (June 21, 2021)
```

Comment:
Thanks, I forgot about the NEWS.md :-)
Since we want to add a new release on PyPI, we will have two new items (corresponding to the two releases). I'll remove the older date but keep the two separate, if that is fine.

Reply:
makes sense